docs/index.md

```diff
@@ -1,3 +1,17 @@
+# Charmed Wazuh Dashboard Documentation
+The [Wazuh Dashboard](https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/index.html)
+is a component for analyzing and visualizing security data data based on [OpenSearch](http://opensearch.org/). Consequentily,
+this charm is a Charmed OpenSearch Dashboards fork.
 
+For the complete documentation, refer to [upstream](https://charmhub.io/opensearch-dashboards).
 
-# Contents
\ No newline at end of file
+## Project and community
+
+The Wazuh Dashboard Operator is a member of the Ubuntu family. It's an open-source project that warmly welcomes community projects, contributions, suggestions, fixes, and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
+- [Contribute](https://github.com/canonical/opensearch-operator/blob/2/edge/CONTRIBUTING.md)
+
+Thinking about using the Wazuh Server Operator for your next project? [Get in touch](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)!
```

Comment:
Did you mean to say "Server" here, or should it say Dashboard?

Reply:
Fixed